ops/_private/yaml.py

```diff
@@ -16,18 +16,23 @@
 
 from typing import Any, Optional, TextIO, Union
 
+import opentelemetry.trace
 import yaml
 
+tracer = opentelemetry.trace.get_tracer(__name__)
+
 # Use C speedups if available
 _safe_loader = getattr(yaml, 'CSafeLoader', yaml.SafeLoader)
 _safe_dumper = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)
 
 
+@tracer.start_as_current_span('ops.yaml.safe_load')  # type: ignore
```

Comment:
Why do we want to trace internal calls like this?

Reply:
dropped